easybuild/easyconfigs/s/Samcef/Samcef-17.0-03.eb

```diff
@@ -0,0 +1,14 @@
+name = 'Samcef'
+version = '17.0-03'
+
+homepage = 'http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/'
+description = """FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and
+ thermal applications."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# licensed software, no source URls for automatic download
+# note: unpacking requires that 7z (p7zip) is available
+sources = ['SamcefV17.0-03.iso']
```

Comment:
aah, good point, I'll get that fixed, thanks @fgeorgatos!

@wpoely86: -1 for you ;)